harness-ractor/harness.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+require_relative '../harness/harness-common'
+
+Warning[:experimental] = false
+ENV["YJIT_BENCH_RACTOR_HARNESS"] = "1"
```

Comment:
Since this is a new env variable, how about following the new name of this repo? (it will be harder to change it later, but it's easy now)
```suggestion
ENV["RUBY_BENCH_RACTOR_HARNESS"] = "1"
```
And the same for `YJIT_BENCH_RACTORS` below.